EcoSoapBank/Network Controller/AppQuerier.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  AppQuerier.swift
+//  EcoSoapBank
+//
+//  Created by Jon Bash on 2020-08-20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+
+typealias ResultHandler<T> = (Result<T, Error>) -> Void
+
+
+enum AppQueryError: Error {
+    case noToken
+    case other(Error)
+    case unimplemented
+    case unknown
+}
+
+
+class AppQuerier {
```

Comment:
I think this file was deleted in Shawn's PR; would be worth deleting it in yours so it's not a back & forth deletion game. 😅

Reply:
This file hasn't been in my project since I merged from master after Shawns last PR. I don't know why this is on here :/

Reply:
Deleted.